02week/bio/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="css/style.css">
+    <meta charset="utf-8">
+    <title>My Bio</title>
+  </head>
+  <body>
+    <main>
+      <header>
+        <h1>Welcome to my bio page!</h1><hr>
+      </header>
+    </main>
+    <div>
+      <section class ="profile">
+        <h1>Profile</h1><hr>
+          <img src="./img/IMG_1623.jpg" alt="me"/>
+          <p class ="ital">Phone: 405-686-8240<br>
+            E-mail: peanut61313@gmail.com<br>
+            Address: 7117 Woodhollow Dr.
+          </p>
+          <a href="http://127.0.0.1:3000/01week/resume/index.html">
```

Comment:
this link does not need to have the server (all the numbers part of it). This may have worked while you were running live server on atom, but wont work from other computers (: